dist/iso.js

```diff
@@ -101,6 +112,36 @@ var Iso = (function () {
       return render;
     })()
   }], [{
+    key: 'encode',
+    value: (function () {
+      function encode(str) {
+        var val = toString(str);
+
+        if (rEnc.test(val)) {
+          return val.replace(rLt, '&lt;').replace(rGt, '&gt;');
+        }
+
+        return val;
+      }
+
+      return encode;
+    })()
```

Comment:
i'm very confused as to why this is an IIFE instead of just an inline function reference